google/cloud/testing_util/opentelemetry_matchers.h

```diff
@@ -150,6 +150,13 @@ MATCHER_P(SpanWithParent, span,
   return actual == span->GetContext().span_id();
 }
 
+MATCHER(SpanIsRoot, " is root span") {
+  auto const& actual =
+      (arg->GetParentSpanId() == opentelemetry::trace::SpanId());
```

Comment:
nit: unnecessary parentheses, and we take `bool`s by value, not ref

```cc
  auto const actual = arg->GetParentSpanId() == opentelemetry::trace::SpanId();
```

Reply:
Done